### ISSUE TITLE
Extract command breaks translation files

### ISSUE THREAD

**Issue by emzet** [ngneat/transloco-keys-manager#43]
## I'm submitting a...
```
[X] Bug report
```

## Current behavior
When runing `transloco-keys-manager extract` with this template markup, it outputs message in console and breaks existing translation JSON files. Same problem as here https://github.com/ngneat/transloco-keys-manager/issues/32. `[inlineSVG]` is directive from library https://github.com/arkon/ng-inline-svg

**Bad**
```
<span class="my-span-class another-class also-span-class"
  [inlineSVG]="'some/path/to/image.svg'"
  data-e2e="DataAttributeValue"
  matTooltipClass="my-tooltip-class"
  matTooltip="{{ 'MY_MODULE.MY_SUBMODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}">
</span>
```

**Good** (_no message in console, extract successful_)
```
<span class="my-span-class another-class also-span-class"
  matTooltip="{{ 'MY_MODULE.MY_SUBMODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}"
  [inlineSVG]="'some/path/to/image.svg'"
  data-e2e="DataAttributeValue"
  matTooltipClass="my-tooltip-class">
</span>
```

## Expected behavior
Extract translation keys without breaking the existing translation files.

## Minimal reproduction of the problem with instructions
For example add to any template:
```
<span class="my-span-class another-class also-span-class"
  [inlineSVG]="'some/path/to/image.svg'"
  data-e2e="DataAttributeValue"
  matTooltipClass="my-tooltip-class"
  matTooltip="{{ 'MY_MODULE.MY_SUBMODULE.MY_COMPONENT.MY_TRANSLATION_KEY' | transloco }}">
</span>
```

Then run from console `transloco-keys-manager extract` and see message in console and changes on your existing translation JSON files (defined in `transloco.config.js`).

## What is the motivation / use case for changing the behavior?
To work `transloco-keys-manager extract` as intended and without breaking the existing translation files.

## Environment

```
Angular: 9.1.11
Angular CLI: 9.1.8
Transloco: 2.17.2
Transloco keys manager: 2.1.0
ng-inline-svg: 10.1.0
 
For Tooling issues:
- Node: 14.4.0
- Platform: Windows 10
```

**Comment by shaharkazaz** [comment]
@emzet Can you please provide the console error?

**Comment by emzet** [comment]
@shaharkazaz same as here https://github.com/ngneat/transloco-keys-manager/issues/32

**Comment by shaharkazaz** [comment]
@emzet should be fixed in v2.4.1
